docs/docsource/installation/configure/create_an_initial_access_key.rst

```diff
@@ -1,32 +1,35 @@
-Create a First AccessKey Using vaultclient
-==========================================
+.. _Create an Initial Access Key:
 
-The first set of credentials of an account is created using
-``vaultclient`` and admin credentials. Later on, account credentials
-can be used with AWS API to create users and policies.
+Create an Initial Access Key
+============================
+
+The first set of credentials for an account is created using ``vaultclient`` and
+admin credentials. Use these account credentials with the AWS API to create
+users and policies.
 
 Prerequisites
 -------------
 
 - The data service is deployed
-- Admin credentials 
-- Admin routes
+- Admin credentials are available 
+- Admin routes are set
```

Comment:
What does this mean?
How will end-user ensure that  Admin Routes are available? And what are admin routes over here?